libs/store_helper/user/ebpf_registry_helper.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "ebpf_store_helper.h"
+
+#define REG_CREATE_FLAGS (KEY_WRITE | DELETE | KEY_READ)
+#define REG_OPEN_FLAGS (DELETE | KEY_READ)
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    void
+    ebpf_close_registry_key(ebpf_store_key_t key);
+
+    _Must_inspect_result_ ebpf_result_t
+    ebpf_write_registry_value_binary(
```

Comment:
nit: is it possible to merge this and the KM registry_helper.h into a single header file (but still have 2 different implementations / .c files)?

Reply:
UM & KM have different functions (i.e. UM implements more helpers), so that'd require using preprocessor vars...